crates/weaver_resolver/data/registry-test-4-events/registry/browser-ref-event.yaml

```diff
@@ -0,0 +1,52 @@
+groups:
+  - id: browser.pageview
+    name: browser.pageview
+    type: event
+    brief: >
+      This event represents an occurrence of a page view.
+    attributes:
+      - ref: session.id
+      - ref: browser.platform
+    body:
+      fields:
+        - ref: http.url
```

Comment:
These are NOT attributes in the same sense as the registry attributes. The recommendation is that field names should be as short as possible, they are also unique to the specifically named event (ie. they are not shared with ANYTHING else).

Reply:
I'll bring this up in the Event Sig today, but this both predates any of the embed discussions and is a different scenario as a body field is not an attribute.

This referencing is mostly to avoid repeating and re-defining the same thing over and over again, for example things like `http.url` may be present in an event body as multiple fields but would need to have different names (alias) within the body. `url`, `referrer`, etc (the names will depend on the event definition).

Reply:
Outcome from Events SIG: While we have a strong `desire` to have the ability to "ref" an existing attribute to avoid duplication, we don't have a strong `need` at this point, so we all agreed that I'm ok to remove this concept from weaver for this initial version.

We strongly "suspect" that once event yaml starts to get defined there will be "questions" raised about why all the duplication -- which is what we (and me) where trying to avoid.